src/components/Day.vue

```diff
@@ -195,13 +195,16 @@ export default {
 
     isHighlighted() {
       if (!this.checkIn) return false;
-      if (this.isDisabled) return false;
+      if (this.isAfterMaxCheckOutDate) return false;
       if (this.choosingCheckOut) return this.isDayInRange(this.date, [this.checkIn, this.hoveringDate]);
       return this.isDayInRange(this.date, [this.checkIn, this.checkOut]);
     },
 
     isCurrentCheckOutCandidate() {
-      return this.choosingCheckOut && this.isHighlighted && this.isHovered && !this.isCheckInDay;
+      if (!this.choosingCheckOut) return null;
```

Comment:
Should returns false right?

Reply:
Well, I guess `null` was more appropriate here, since there is no such thing as a current checkout candidate when the user is not chossing a checkout date.

Reply:
Fixed :heavy_check_mark: